activesupport/lib/active_support/core_ext/object/blank.rb

```diff
@@ -113,9 +115,11 @@ class String
   #   "\t\n\r".blank? # => true
   #   ' blah '.blank? # => false
   #
-  # Unicode whitespace is supported:
+  # Unicode whitespace and zero width space are supported:
```

Comment:
This is just my opinion, but I would consider leaving this as just `Unicode whitespace is supported:` without mentioning zero-width spaces.  The reason is: I think most people would already consider zero-width spaces to be "Unicode whitespace", and we already aren't mentioning no-break spaces (`\u00A0`) explicitly.

Reply:
I agree.